packages/trpc/client/trpc.ts

```diff
@@ -7,8 +7,7 @@ import type { AppRouter } from "../server/router";
 export const trpc = createTRPCClient<AppRouter>({
 	links: [
 		httpBatchLink({
-			// Don't use the env package here, as this is in the client bundle
-			url: import.meta.env.VITE_PUBLIC_API_URL ?? "/trpc",
+			url: "/trpc",
```

Comment:
This should use the environment variable. We want to be able to change where the link is pulling from, it might not always be `/trpc`.

```suggestion
			url: import.meta.env.VITE_PUBLIC_API_URL ?? "/trpc",
```

Reply:
Addressed: the url is now  `${getBaseUrl()}/trpc`, where `getBaseURL` returns `VITE_PUBLIC_API_URL` in prod and `""` in dev. The dev url must be relative to avoid bypassing the proxy. Changing `/trpc` in the template literal will update behavior for both prod and dev without having to adjust their urls in the environment variables.